consensus/src/sync/sync_queue.rs

```diff
@@ -22,7 +20,7 @@ use crate::sync::peer_list::PeerListIndex;
 
 #[pin_project]
 #[derive(Debug)]
-struct OrderWrapper<TId, TOutput> {
+struct OrderWrapper<TId: Clone, TOutput> {
```

Comment:
Why does this need to be `Clone`? I cannot find it as new requirement.

Reply:
True, removed it again.